content/en/account_management/rbac/_index.md

```diff
@@ -41,15 +41,13 @@ Datadog Read Only Role
 
 ## Custom roles
 
-<div class="alert alert-warning">
-Creating and modifying custom roles is an opt-in Enterprise feature. <a href="/help">Contact Datadog support</a> to get it enabled for your account.
-</div>
+Manage your custom roles through the Datadog application, the [Datadog Role API][6], or SAML directly. Find below how to create, update, delete a role. See the [Datadog Role Permissions][7] documentation for more information about available permissions. Only users with the Access Management permission can create or edit roles in Datadog.
 
-Manage your custom roles through the Datadog application, the [Datadog Role API][6], or SAML directly. Find below how to create, update, delete a role. See the [Datadog Role permissions][7] documentation for more information about available permissions. Only users with the Access Management permission can create or edit roles in Datadog.
+### Enable custom roles
```

Comment:
Adding this section since there will be a way to add custom roles directly through the UI. Left a `TODO` below to actually write the description.

Reply:
Some wordsmithing on this section - 
Custom roles gives your org the ability to create new roles with unique permission sets. Manage your custom roles through the Datadog application, the [Datadog Role API][6], or SAML directly. Find below how to create, update, delete a role. See the [Datadog Role permissions][7] documentation for more information about available permissions. Only users with the "User Access Manage" permission can create or edit roles in Datadog.

Reply:
Our style guide prefers "Datadog site" to "Datadog application". We are also supposed to avoid the word "documentation" in favor of using the title of the page. In other places where we use the display name of permissions, it doesn't have quotes around it, and I'd like to keep that consistent. I did take the other wording suggestions here, though. Thanks!